File: Week_01/id_17/LeetCode_26_17.c
int removeDuplicates(int* nums, int numsSize){
    
    int cnt = 1;
    
    if (nums == NULL || numsSize == 0){
        return 0;
    }
    
    for (int i = 1; i < numsSize; i++){
        if (nums[i] != nums[i - 1])     
            nums[cnt++] = nums[i];
    }
    return cnt;
}

File: Week_01/id_17/LeetCode_26_17.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) <= 1:
            return len(nums)
        cnt = 0
        for idx in range(0,len(nums)):
            if nums[idx] != nums[idx - 1]:
                nums[cnt] = nums[idx]
                cnt += 1
        return cnt
